### ISSUE TITLE
CalDav Support

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#690]
Originally reported on Google Code with ID 420

```
Any chance that this will one day feature CalDav support?
I'd like to encapsulate all data on the one server (ie.. not have to use Google), and
can do so with an 
open source PHP calDav server, but as far as I can tell FullCalendar does not support
CalDav feeds?
```

Reported by `tnbenson` on 2010-03-24 21:41:18

Imported with **26** stars.

**Comment by 12345t6zhnjko987z6t5redsxcdfghuio0pp9i8** [comment]
Why not use sabre caldav : http://sabre.io/dav/caldav/ ?

**Comment by espen** [comment]
sabre/dav is just a webdav server framework.

CalDavZap at https://www.inf-it.com/open-source/clients/caldavzap/ is a javascript CalDav client that uses FullCalendar in the demo: https://www.inf-it.com/caldavzap/

There is also https://github.com/mozilla-b2g/caldav from Mozilla.

**Comment by TomFernandes** [comment]
Also interested in this feature. We use ownCloud-calendars (CalDAV-based) at work for different purpose. One purpose is to give a team an overview of the holiday situation. If we could use this calendar as storage for fullcalendar, this would make live easy, as we could have all of our calendars at a central location (for easy syncing to different calender-clients).

**Comment by NachoVazquez** [comment]
This is exactly what I'm looking for. My company implemented a ``CalDAV`` server in ASP.NET Core and right now I'm kind of implementing a ``CalDAV`` Client with ``fullcalendar`` but from my experience developing the server I know it is hard work. I'll be eager to contribute if this feature start been developed.

**Comment by mpmullally** [comment]
This would be the best feature, there are very few `CalDav` viewers. WHen I subscribe to my icloud calendar in google calendar the times are in UTC and I can't change them to eastern.

**Comment by metasyn** [comment]
I agree - this would be an incredible front end - its very hard to find `CalDAV` on the web that are really up to date. The closest I've found is [AgenDAV](https://github.com/agendav/agendav) which does not have the same level of functionality that fullcalendar has. Another example might be the [calendar app](https://github.com/nextcloud/calendar) that nextcloud uses - which uses fullcalendar but *also* had integration with `CalDAV`.

**Comment by metasyn** [comment]
https://github.com/modoboa/modoboa-radicale/blob/master/modoboa_radicale/backends/caldav_.py

Came across this today, which looks like it does the majority of the heavy lifting of getting caldav to be re-hosted as an api endpoint that is compatible as an event source for fullcalendar

**Comment by mikaelcom** [comment]
> sabre/dav is just a webdav server framework.
> 
> CalDavZap at https://www.inf-it.com/open-source/clients/caldavzap/ is a javascript CalDav client that uses FullCalendar in the demo: https://www.inf-it.com/caldavzap/
> 
> There is also https://github.com/mozilla-b2g/caldav from Mozilla.

I'm using Sabre and AFAIK in PHP it's perfetly fitted to handle both CALDav and CardDav which works like a charm, @arshaw you should give it a try if you're looking for a PHP/MySQL solution.
I'm using [Radicale ](https://radicale.org/3.0.html) since a while to replace Google but it stops from working for no reason, I have to restart sometimes.
I'm working on a full PHP platform integrating both Contacts and Calendar with [Sabre](https://sabre.io/) solutions in order only use this solution, replacing Radicale in the end.

**Comment by escoand** [comment]
I've created a caldav (and year view) plugin for my own use. Maybe it's helpful for somebody else too: https://github.com/escoand/fullcalendar-plugins

**Comment by Eviepayne** [comment]
I'm surprised this doesn't get more attention.
This is literally the plugin I've been waiting for months now.
